06-Plotting-Time-Series-Rasters-in-R.Rmd

```diff
@@ -136,31 +136,37 @@ plot(NDVI_stack,
 
 ``` 
 
-In this lesson we will refine our plot using `levelplot` from the `rasterVis`
-library. The syntax is similar to plot.
+Note: The metadata for this NDVI data have a scale factor: 10,000. A scale
+factor is commonly used in larger datasets to remove decimals. {: .notice2}
```

Comment:
This doesn't make sense as written.

Note: The range of values for NDVI is 0-1. However, the data stored in our raster ranges from 0 - 10,000. If we view the metadata for the original `.tif` files, we will see a scale factor of 10,000 has been applied to remove decimal places. A scale factor allows us to store the NDVI data as an integer (no decimals) rather than a  floating point format (decimals). This keeps the raster file size smaller.
{ : .notice }